jax.errors.SimplifiedTraceback: For simplicity, JAX has removed its internal frames from the traceback of the following exception. Set JAX_TRACEBACK_FILTERING=off to include these.
The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "/Users/akshayjacobthomas/Documents/GitHub/DiffMicromehanics/surrogate/main.py", line 40, in <module>
    app.run(main)
  File "/Users/akshayjacobthomas/anaconda3/envs/jax_trial/lib/python3.9/site-packages/absl/app.py", line 308, in run
    _run_main(main, args)
  File "/Users/akshayjacobthomas/anaconda3/envs/jax_trial/lib/python3.9/site-packages/absl/app.py", line 254, in _run_main
    sys.exit(main(argv))
  File "/Users/akshayjacobthomas/Documents/GitHub/DiffMicromehanics/surrogate/main.py", line 32, in main
    train.train_and_evaluate(FLAGS.config, FLAGS.workdir)
  File "/Users/akshayjacobthomas/Documents/GitHub/DiffMicromehanics/surrogate/train.py", line 40, in train_and_evaluate
    model = models.MICRO_SURROGATE(config)
  File "/Users/akshayjacobthomas/Documents/GitHub/DiffMicromehanics/surrogate/models.py", line 20, in __init__
    super().__init__(config)
  File "/Users/akshayjacobthomas/Documents/GitHub/DiffMicromehanics/surrogate/NN_surrogate/models.py", line 109, in __init__
    self.state = _create_train_state(config)
  File "/Users/akshayjacobthomas/Documents/GitHub/DiffMicromehanics/surrogate/NN_surrogate/models.py", line 87, in _create_train_state
    params = arch.init(random.PRNGKey(config.seed), x)
  File "/Users/akshayjacobthomas/Documents/GitHub/DiffMicromehanics/surrogate/NN_surrogate/archs.py", line 82, in __call__
    for dim in self.hidden_dims:
AttributeError: "Mlp" object has no attribute "hidden_dims".